src/stylesheets/notification.less

```diff
@@ -6,14 +6,8 @@
 
     .sk-notification {
         overflow: hidden;
-
-        height: @notification-height;
```

Comment:
Are `@notification-height` and `@long-notification-height` still used somewhere? Can we delete those vars?